apps/avifenc.c

```diff
@@ -230,6 +239,10 @@ static void syntaxLong(void)
     printf("    --ignore-profile,--ignore-icc     : If the input file contains an embedded color profile, ignore it (no-op if absent)\n");
 #if defined(AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION)
     printf("    --ignore-gain-map                 : If the input file contains an embedded gain map, ignore it (no-op if absent)\n");
+    printf("    --qgainmap Q                      : Set quality for the gain map (%d-%d, where %d is lossless)\n",
```

Comment:
Guard with #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)

Reply:
AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION implies AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP.
I changed it to use AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION everywhere in this file for simplicity/consistency and added some comments.